File: spec/factories/movie.rb
# frozen_string_literal: true

FactoryBot.define do
  factory :movie do
    name { 'user name' }
    description { ' movie description' }
    thumbnail { 'movie_thumbnail.png' }
    director { 'movie director' }
    actor { 'actor1, actor2' }
    year { 2022 }
  end
end

File: client/src/app/shared/movie/domain/movie.ts
export interface MoviePreview {
  id: number;
  name: string;
  thumbnail: string;
  description: string;
}

export interface Movie {
  id: number;
  name: string;
  thumbnail: string;
  description: string;
  director: string;
  actor: string;
  year: number;
  reviews: string[];
}
